fuzzers/047-hclk-ioi-pips/Makefile

```diff
@@ -0,0 +1,45 @@
+export FUZDIR=$(shell pwd)
+PIP_TYPE?=hclk_ioi3
+PIPLIST_TCL=$(FUZDIR)/hclk_ioi3_pip_list.tcl
+TODO_RE=".*"
```

Comment:
Change this TODO_RE to filter only include pips we are looking for right now, and we can land this PR.

Reply:
Added an EXCLUDE_RE and also min-progress to be able to land this PR.
The bits used by Artix will be solved by the time we get to the min progress trigger.